index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script src="app.js"></script>
+<body>
+<style type="text/css"></style>
+<h2> <span class="label label-default">Simple Application that generates passwords with good complexity</span></h2>
+<br>
+<div class="well" id="pass" >
+	This is where the password will be generated !<br> </div> 
+
+<div class="alert alert-info" role="alert">
+	To generate a password click on <a  onclick="gen()">GENERATE
+</div>
+
+<script>
+function gen() {
```

Comment:
keep the html clean ;) 
why didn't you include this function in a typescript file ?